### PR TITLE
Recovery

### PR DESCRIPTION
**Pull request by INR4GE**

### CODE REVIEW

File: typeDefs.js [hunk 1]
```diff
@@ -288,6 +288,8 @@ const typeDefs = gql`
     updateUserByToken(user: UserInput!): User
     confirmUser(token: String!): Boolean
     recoverUser(email: String!, language: Int!): Boolean
+    changePassword(password: String!, token: String!): Boolean
```
Comment (Alan18081) [inline typeDefs.js:291]: Please, rename it to the `resetPassword` to not match with the mutation `changePassword` that you can do from the user page

File: typeDefs.js [hunk 2]
```diff
@@ -288,6 +288,8 @@ const typeDefs = gql`
...
+    checkIfExists(token: String!): Boolean
```
Comment (Alan18081) [inline typeDefs.js:292]: Rename it to the `checkIfTokenIsValid`, it is much more explicit

File: typeDefs.js (hunk 1 above)
Comment (INR4GE) [reply typeDefs.js]: changed

File: typeDefs.js (hunk 2 above)
Comment (INR4GE) [reply typeDefs.js]: renamed

**Pull request merged**